exporter/fileexporter/metadata.yaml

```diff
@@ -7,3 +7,41 @@ status:
   distributions: [core, contrib, observiq, splunk, sumo, aws]
   codeowners:
     active: [atingchen]
+config:
```

Comment:
it would be good to pick another exporter where there is a heavy reuse of shared config objects, like the HTTPSettings. How would it be handled in the schema?